Composer/packages/client/src/pages/design/PropertyEditor.tsx

```diff
@@ -28,11 +28,11 @@ const PropertyEditor: React.FC = () => {
   const { shellApi, ...shellData } = useShellApi();
   const { currentDialog, data: formData = {}, focusPath, focusedSteps, schemas } = shellData;
 
-  const currentWidth = shellData?.userSettings?.propertyEditorWidth || 400;
+  // const currentWidth = shellData?.userSettings?.propertyEditorWidth || 400;
```

Comment:
what's the plan with this commented out stuff?

Reply:
Dang - missed removing that one too :-S Will clean up.